docs/api/ReactWrapper/childAt.md

```diff
@@ -0,0 +1,28 @@
+# `.childAt(index) => ReactWrapper`
```

Comment:
note: you will want to include both of these pages in the `/docs/README.md` page, as well as including links to them in the `/docs/api/shallow.md` and `/docs/api/mount.md` pages.

Thanks!